frontend/views/components/ProfileCard.vue

```diff
@@ -0,0 +1,268 @@
+          tag='button'
+          @click='openModal("incomeDetails")'
+          data-test='linkEditPayment'
+        ) Edit payment info
+
+      .c-add-payment(v-else-if='isSelf')
+        i18n.has-text-1(tag='p') Help other users send monthly contributions your way by adding your payment information.
+        i18n.button.c-add-payment-button(
+          tag='button'
+          @click='openModal("incomeDetails")'
+          data-test='buttonAddPayment'
+        ) Add payment information
+
+      .buttons(v-if='!isSelf')
+        i18n.button.is-outlined.is-small(
+          tag='button'
+          @click='sendMessage'
+        ) Send message
+
+        i18n.button.is-outlined.is-small(
+          tag='button'
+          @click='openModal("RemoveMember", { username })'
+        ) Remove member
+      modal-close.c-close(:aria-label='L("Close profile")' @close='toggleTooltip')
+</template>
+
+<script>
+import AvatarUser from '@components/AvatarUser.vue'
+import UserName from '@components/UserName.vue'
+import Tooltip from '@components/Tooltip.vue'
+import ModalClose from '@components/modal/ModalClose.vue'
+import { OPEN_MODAL } from '@utils/events.js'
+import sbp from '~/shared/sbp.js'
+import { mapGetters } from 'vuex'
+
+export default {
+  name: 'ProfileCard',
+  props: {
+    username: String,
+    isSelf: {
+      type: Boolean,
+      default: false
+    },
+    direction: {
+      type: String,
+      validator: (value) => ['left', 'top-left'].includes(value),
+      default: 'left'
+    }
+  },
+  components: {
+    AvatarUser,
+    ModalClose,
+    UserName,
+    Tooltip
+  },
+  data () {
+    return {
+      // Temp
+      fakeStore: {
+        paymentMethods: [{
+          name: 'Paypal',
+          address: 'maggie@mail.com'
+        }, {
+          name: 'Bitcoin',
+          address: '1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2'
+        }, {
+          name: 'Venmo',
+          address: '@maggieb'
+        }, {
+          name: 'MBWAY',
+          address: '+351913000330'
+        }]
+      }
+    }
+  },
+  methods: {
+    openModal (modal, props) {
+      this.toggleTooltip()
+      sbp('okTurtles.events/emit', OPEN_MODAL, modal, props)
+    },
+    toggleTooltip () {
+      this.$refs.tooltip.toggle()
+    },
+    sendMessage () {
+      console.log('To do: implement send message')
+    },
+    hasIncomeDetails () {
+      return !!this.ourGroupProfile.incomeDetailsType
+    },
+    receivingMonetary () {
+      return !!this.ourContributionSummary.receivingMonetary
+    }
+  },
+  watch: {},
+  computed: {
+    ...mapGetters([
+      'ourUsername',
+      'ourGroupProfile',
+      'groupSettings',
+      'groupMembersCount',
+      'groupProfile',
+      'groupProfiles',
+      'groupMincomeFormatted',
+      'globalProfile',
+      'ourContributionSummary'
+    ]),
+    profile () {
+      return this.$store.getters.globalProfile(this.username)
+    },
+    isActiveGroupMember () {
+      return !!this.groupProfiles[this.username]
```

Comment:
FYI in #900 (Payments) this is going to change, there will be a `.status` property on the groupProfile that you can check, e.g. `return this.groupProfiles[this.username].status === INVITE_STATUS.VALID`.

So either let's wait for that to be merged and update this, and/or add a `// TODO:` comment to that effect here to not forget to update this.